aspnetcore/index.md

```diff
@@ -16,66 +16,47 @@ uid: index
-
+* Build HTTP services that reach a broad range of clients, including browsers and mobile devices.  Support for [multiple data formats and content negotiation](mvc/models/formatting.md) is built-in. ASP.NET Core is an ideal platform for building web APIs and RESTful apps on .NET Core. See [Building web APIs](tutorials/index.md#building-web-apis).
 * You can create well-factored and testable web apps that follow the Model-View-Controller (MVC) pattern. See [MVC](mvc/index.md) and [Testing](testing/index.md).
-
-* [Razor](http://www.asp.net/web-pages/overview/getting-started/introducing-razor-syntax-c) provides a productive language to create [Views](mvc/views/index.md)
-
-* [Tag Helpers](mvc/views/tag-helpers/intro.md) enable server-side code to participate in creating and rendering HTML elements in Razor files
-
-* [Model Binding](mvc/models/model-binding.md) automatically maps data from HTTP requests to action method parameters
-
-* [Model Validation](mvc/models/validation.md) automatically performs client and server side validation
+* [Razor](http://www.asp.net/web-pages/overview/getting-started/introducing-razor-syntax-c) provides a productive language to create [Views](mvc/views/index.md).
+* [Tag Helpers](mvc/views/tag-helpers/intro.md) enable server-side code to participate in creating and rendering HTML elements in Razor files.
+* [Model Binding](mvc/models/model-binding.md) automatically maps data from HTTP requests to action method parameters.
+* [Model Validation](mvc/models/validation.md) automatically performs client and server side validation.
 
 ## Client-side development
 
-ASP.NET Core is designed to integrate seamlessly with a variety of client-side frameworks, including [AngularJS](client-side/angular.md), [KnockoutJS](client-side/knockout.md) and [Bootstrap](client-side/bootstrap.md). See [Client-Side Development](client-side/index.md) for more details.
-
-## Next steps
-
-For getting-started tutorials, see [ASP.NET Core Tutorials](tutorials/index.md)
-
-For in-depth introduction to ASP.NET Core concepts and architecture, see [ASP.NET Core Fundamentals](fundamentals/index.md).
+ASP.NET Core is designed to integrate seamlessly with a variety of client-side frameworks, including [AngularJS](client-side/angular.md), <!-- KO is no longer featured --> [KnockoutJS](client-side/knockout.md) and [Bootstrap](client-side/bootstrap.md). See [Client-Side Development](client-side/index.md) for more details.
 
-An ASP.NET Core app can use the .NET Core or .NET Framework runtime. For more information, see [Choosing between .NET Core and .NET Framework](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server).
+## For more information
```

Comment:
We never title this section "For more information" -- usually it's Next steps, though sometimes we use Additional resources. Here you could put "For more information, see the following resources" under a Next steps title.